acceptance-tests/src/test/java/gov/di_ipv_drivingpermit/pages/WelshLangDrivingLicencePageObject.java

```diff
@@ -0,0 +1,1015 @@
+        Assert.assertEquals(expTitle, actualTitle);
+        LOGGER.info(actualTitle);
+    }
+
+    public void ErrorPageTitleDVLA() {
+        String actualTitle = Driver.get().getTitle();
+        String expTitle =
+                "Gwall: Rhowch eich manylion yn union fel maent yn ymddangos ar eich trwydded yrru – – GOV.UK";
+        Assert.assertEquals(expTitle, actualTitle);
+        LOGGER.info(actualTitle);
+    }
+
+    public void dvlaPageHeading() {
+        String expectedText =
+                "Rhowch eich manylion yn union fel maent yn ymddangos ar eich trwydded yrru";
+        String actualText = dvlaPageHeadingWelsh.getText();
+        Assert.assertEquals(expectedText, actualText);
+        LOGGER.info(actualText);
+    }
+
+    public void dvlaProveYourIdentitySentence() {
+        String expectedText =
+                "Os nad oes gennych drwydded yrru y DU neu os na allwch gofio'ch manylion, gallwch brofi pwy ydych chi mewn ffordd arall yn lle.";
+        String actualText = proveIdentityTextWelsh.getText();
+        Assert.assertEquals(expectedText, actualText);
+        LOGGER.info(actualText);
+    }
+
+    public void checkYourDetailsSentence() {
+        String expectedText =
+                "Gwiriwch bod eich manylion yn paru gyda beth sydd ar eich trwydded yrru y DU";
+        String actualText = checkYourDetailTextWelsh.getText();
+        Assert.assertEquals(expectedText, actualText);
+        LOGGER.info(actualText);
+    }
+
+    public void erroeWord() {
+        String expectedText = "Gwall";
+        String actualText = errorTextWelsh.getText();
+        Assert.assertEquals(expectedText, actualText);
+        if (Driver.get().getPageSource().contains("Gwall")) {
+            LOGGER.info(actualText);
+        }
+    }
+
+    public void weCouldNotFindDetailsSentence() {
+        String expectedText = "Nid oeddem yn gallu dod o hyd i'ch manylion";
+        String actualText = weWereUnableToFind.getText();
+        Assert.assertEquals(expectedText, actualText);
+        LOGGER.info(actualText);
+    }
+
+    public void youWillBeAbleToFindSentence() {
+        String expectedText = "Roedd yna broblem wrth i ni wirio eich manylion gyda'r DVLA.";
+        String actualText = thereWasAProblemDVLA.getText();
+        Assert.assertEquals(expectedText, actualText);
+        LOGGER.info(actualText);
+    }
+
+    public void youWillBeAbleToFindSentenceDVA() {
+        String expectedText = "Roedd yna broblem wrth i ni wirio eich manylion gyda'r DVA.";
+        String actualText = checkBannerDVAMsg.getText();
+        Assert.assertEquals(expectedText, actualText);
+        LOGGER.info(actualText);
+    }
+
+    public void thereIsaProblemSentence() {
+        if (Driver.get()
+                .getPageSource()
+                .contains(
+                        "Ni fyddwch yn gallu newid eich manylion eto os byddwch yn gwneud camgymeriad")) {
+            LOGGER.info(
```

Comment:
Should this not be an assert

Reply:
Changed to Assert.assertEquals(expectedText, actualText)